elide-core/src/main/java/com/yahoo/elide/core/DataStoreTransaction.java

```diff
@@ -139,4 +139,28 @@ default void flush() {
                                                           Optional<Sorting> sorting, Optional<Pagination> pagination) {
         return collection;
     }
+
+    default <T> Object getRelation(
+            Object entity,
+            RelationshipType relationshipType,
+            String relationName,
+            Class<T> relationClass,
+            EntityDictionary dictionary,
+            Set<Predicate> filters
+    ) {
+        return PersistentResource.getValue(entity, relationName, dictionary);
```

Comment:
I don't think we're gonna make that assumption now. We haven't ready yet to make proxy-ize everything. Designing proxies for other data stores might be a lot of work. 

I'm trying to avoid involving PersistentResource into transactions, they would be pointing to each other and logic may entangle.